### ISSUE TITLE
New release?

### ISSUE THREAD

**Issue by yann-morin-1998** [pocmo/Yaaic#232]
Hello!

There has been no commit in the past year or so. I do understand
that people go other routes as time passes, and that is perfectly fine.

However, the current version on fdroid is still using vereskion 1.1
(which is more than 2 years ago) and is marked as broken.

Can we come to a conclusion and tag a release so we can at least
update to the current code? We need a tag, to be sure the version
code is stable andnot reused later.

Thank you! :-)

Regards,
Yann E. MORIN.

**Comment by pocmo** [comment]
Yeah, I'm sorry for not working on Yaaic more often. Unfortunately I do not have as much time as I had when I was developing Yaaic.

I do not really consider the current _master_ version stable. It's in-between a code and UI refactoring that I haven't completed. The platform and UI guidelines change/evolve definitely quicker than Yaaic - so I'm not even sure I would want to complete *this* version. I still have a bunch of ideas how I would like to build Yaaic - however I do not know how to find the time for a project that mostly takes time away from my family and that I do not use myself currently. :)

**Comment by yann-morin-1998** [comment]
@pocmo: I totally understand that familly is more important. You are most
definitely right to spend your time with them. Lucky you! :-)

As for the code/UI refactoring, I understand it is not complete either. Just tag
it as `2.0alpha0` or whatever, so people understand that it is not meant to be
stable (it is enough for my usecase, anyway). With a tag, we know the version
code will not be reused later, and we can add it to fdroid.

Thank you for Yaaic! It is a very good program that I do use regularly. Thanks ! :-)

Keep well!

Regards,
Yann E. MORIN.

**Comment by kaecy** [comment]
As I recall it was never stable. It's always been buggy for me. But it's the best at what it does. I read in the store the scrolling bug was fixed, it's now stable in my mind. We can finally call this stable.

**Comment by smaragdus** [comment]
I hope this is not a dead project.

**Comment by rzr** [comment]
At least less dead than https://github.com/indrora/Atomic 

Relate to:  https://github.com/abandonware/abandonware.github.io/issues/5

**Comment by smaragdus** [comment]
To this day I haven't found an Android IRC client that is barely usable. Any suggestions?

**Comment by pocmo** [comment]
> To this day I haven't found an Android IRC client that is barely usable.

What's the problem with the available clients?
